services/rest/src/main/java/mil/nga/giat/geowave/service/rest/GeoWaveOperationServiceWrapper.java

```diff
@@ -227,7 +229,10 @@ else if (String.class.isAssignableFrom(type)) {
 					else if (Enum.class.isAssignableFrom(type)) {
 						objValue = Enum.valueOf(
 								(Class<Enum>) type,
-								strValue);
+								strValue.toUpperCase());
+					}
+					else if (CQLFilterOptionProvider.FilterParameter.class.isAssignableFrom(type)) {
```

Comment:
instead of adding `geowave-adapter-vector` and special casing this one parameter that requires a converter, it'd be better to use reflection to find the annotations with a converter and convert them (so it would hypothetically work for any parameter with a converter)